packages-proprietary/tokens/src/components/ams/progress-list.tokens.json

```diff
@@ -1,35 +1,99 @@
 {
   "ams": {
     "progress-list": {
-      "body": {
-        "padding-block-start": { "value": "{ams.space.s}" },
-        "padding-block-end": { "value": "{ams.space.l}" }
-      },
-      "connector": {
-        "background-color": { "value": "{ams.color.separator}" },
-        "inline-size": { "value": "{ams.border.width.m}" }
-      },
-      "indicator": {
-        "padding-inline": { "value": "{ams.space.m}" }
-      },
-      "marker": {
-        "background-color": { "value": "{ams.color.separator}" },
+      "step": {
+        "body": {
+          "padding-block-end": { "value": "{ams.space.l}" },
+          "padding-block-start": { "value": "{ams.space.s}" }
+        },
+        "completed": {
+          "connector": {
+            "background-color": { "value": "{ams.color.feedback.success}" },
+            "border": { "value": "none" },
+            "inline-size": { "value": "2px" }
+          },
+          "marker": {
+            "background-color": { "value": "{ams.color.feedback.success}" },
+            "border": { "value": "none" }
+          }
+        },
+        "connector": {
+          "background-color": { "value": "{ams.color.background}" },
+          "border-color": { "value": "{ams.color.separator}" },
+          "border-style": { "value": "dashed" },
+          "border-width": { "value": "calc({ams.border.width.m} / 2)" }
```

Comment:
Can’t we apply the border to one side instead of halving a token? This also removes the border from the top and bottom, which is what we want.